spring-pulsar/src/main/java/org/springframework/pulsar/listener/DefaultPulsarMessageListenerContainer.java

```diff
@@ -286,6 +299,7 @@ public boolean isLongLived() {
 
 		@Override
 		public void run() {
+			DefaultPulsarMessageListenerContainer.this.listenerConsumerThread = Thread.currentThread();
```

Comment:
Did checkstyle want the `OUTER.this` prefix? I would be in favor of dropping it if checkstyle does not complain.

Reply:
Yes, it was the checkstyle.